packages/@aws-cdk/cloudformation-include/lib/file-utils.ts

```diff
@@ -1,6 +1,14 @@
 import * as fs from 'fs';
+import * as YAML from 'yaml';
+import { shortForms } from './short-form-tags.js';
 
 export function readJsonSync(filePath: string): any {
```

Comment:
Actually, this function is no longer called in production code, right? Can you please move it to somewhere like `test/file-utils.ts`?

Reply:
+1 ;)